src/psycopt2d/load.py

```diff
@@ -0,0 +1,110 @@
+from datetime import date, datetime, timedelta
+from typing import List, Union
+
+import pandas as pd
+from psycopmlutils.loaders import sql_load
+from wasabi import Printer
+
+msg = Printer(timestamp=True)
+
+
+def load_dataset(
+    split_names: Union[List[str], str],
+    drop_if_outcome_before_date: Union[datetime, str],
+    min_lookahead_days: int,
+    datetime_column: str = "pred_timestamp",
+    n_training_samples: Union[None, int] = None,
+) -> pd.DataFrame:
+    """load dataset for t2d.
+
+    Args:
+        split_names (Union[List[str], str]): Names of splits, includes "train", "val",
+            "test".
+        drop_if_outcome_before_date (Union[datetime, str]): Remove patient which have
+            had their outcome before the specified day. Also removed all visits before
+            this date as otherwise the model will learn that all visit before this date
+            does not lead to diabetes.
+            limit the dataset to. Defaults to None.
+        min_lookahead_days (int): Minimum amount of days to required for lookahead.
+            Defined as days from the last days.
+        datetime_column (str, optional): Datetime columns. Defaults to
+            "pred_timestamp".
+        n_training_samples (Union[None, int], optional): Number of training samples to
+    Returns:
+        pd.DataFrame: The filtered dataset
+    """
+    if isinstance(drop_if_outcome_before_date, str):
+        drop_if_outcome_before_date = date.fromisoformat(drop_if_outcome_before_date)
+
+    if isinstance(split_names, list):
+        return pd.concat(
+            [
+                load_dataset(
+                    split,
+                    drop_if_outcome_before_date,
+                    min_lookahead_days,
+                    datetime_column,
+                    n_training_samples,
+                )
+                for split in split_names
+            ],
+        )
+
+    min_lookahead = timedelta(days=min_lookahead_days)
+    sql_table_name = f"psycop_t2d_{split_names}"
```

Comment:
```suggestion
    sql_table_name = f"psycop_t2d_{split_names}"
```
Might want to add this as a parameter. Can be a tiny patch now, or tag me in an issue and I'll get it done ;-)